bool camera_turn_success